spec/unit/plugins/linux/network_spec.rb

```diff
@@ -1219,6 +1219,38 @@
         end
       end
 
+      describe "when there're multipath routes" do
+        let(:linux_ip_route) { "" }
```

Comment:
just to make sure people don't break this in the future, can we add a v4 test, please?